api/Permissions.json

```
@@ -522,7 +522,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "131"
```

@Elchi3 Looking at [bug it is Nightly](https://hg.mozilla.org/mozilla-central/rev/8006c95d029a#l4.12). Searchfox now shows early beta though, so updated at some point.

```suggestion
              "version_added": "preview"
```